src/mutations/item.test.ts

```diff
@@ -0,0 +1,214 @@
+
+    it('multiCopySuccess', async () => {
+      // set random data in cache
+      ITEMS.forEach(item => {
+        const itemKey = buildItemKey(item.id);
+        queryClient.setQueryData(itemKey, Map(item));
+      });
+
+      const response = OK_RESPONSE;
+
+      const endpoints = [
+        {
+          response,
+          method: REQUEST_METHODS.POST,
+          route,
+        }
+      ];
+
+      const mockedMutation = await mockMutation({
+        endpoints,
+        mutation,
+        wrapper,
+      });
+
+      await act(async () => {
+        await mockedMutation.mutate({
+          to,
+          id: copied,
+        });
+        await waitForMutation();
+      });
+
+      // verify cache keys
+      const itemKey = buildItemKey(to);
+      expect(queryClient.getQueryState(itemKey)?.isInvalidated).toBeFalsy();
+    });
+  });
+
+
+  describe(MUTATION_KEYS.MOVE_ITEM, () => {
+
+    const to = ITEMS[0].id;
+    const moved = ITEMS[1].id;
+    const route = `/${buildMoveItemRoute(moved)}`;
+
+    const mutation = () => useMutation(MUTATION_KEYS.MOVE_ITEM);
+
+    it('singleMoveSuccess', async () => {
+      // set random data in cache
+      ITEMS.forEach(item => {
+        const itemKey = buildItemKey(item.id);
+        queryClient.setQueryData(itemKey, Map(item));
+      });
+
+      const response = OK_RESPONSE;
+
+      const endpoints = [
+        {
+          response,
+          method: REQUEST_METHODS.POST,
+          route,
+        }
+      ];
+
+      const mockedMutation = await mockMutation({
+        endpoints,
+        mutation,
+        wrapper,
+      });
+
+      await act(async () => {
+        await mockedMutation.mutate({
+          to,
+          id: moved,
+        });
+        await waitForMutation();
+      });
+
+      const itemKey = buildItemKey(moved);
+      const data = queryClient.getQueryData<Record<Item>>(itemKey);
+
+      // verify cache keys
+      expect(data?.get('path')).toEqual(`${ITEMS[0].path}.${ITEMS[1].path}`);
+    });
+  });
+
+  describe(MUTATION_KEYS.MOVE_ITEMS, () => {
+    const to = ITEMS[0].id;
+    const moved = ITEMS.slice(1).map(x => x.id);
+    const route = `/${buildMoveItemsRoute(moved)}`;
+
+    const mutation = () => useMutation(MUTATION_KEYS.MOVE_ITEMS);
+
+    it('multiMoveSuccess', async () => {
+      // set random data in cache
+      ITEMS.forEach(item => {
+        const itemKey = buildItemKey(item.id);
+        queryClient.setQueryData(itemKey, Map(item));
+      });
+
+      const response = OK_RESPONSE;
+
+      const endpoints = [
+        {
+          response,
+          method: REQUEST_METHODS.POST,
+          route,
+        }
+      ];
+
+      const mockedMutation = await mockMutation({
+        endpoints,
+        mutation,
+        wrapper,
+      });
+
+      await act(async () => {
+        await mockedMutation.mutate({
+          to,
+          id: moved,
+        });
+        await waitForMutation();
+      });
+
+      ITEMS.forEach(item => {
```

Comment:
Why are you looping on `ITEMS`? It would be clearer to loop on `moved` to see if all children are moved correctly, then check the parents' state.